src/Friends/OrganizationsTrait.php

```diff
@@ -22,4 +22,13 @@ public function getOrganizations()
         }
         return $orgs;
     }
+
+    /**
+     * @return SiteOrganizationMemberships|UserOrganizationMemberships
+     * @deprecated 1.0.1 Please use getOrganizationMemberships() instead.
```

Comment:
What does the `1.0.1` mean? Is that the version in which it is deprecated or when it is to be removed or something else?